src/renderer/features/nodeSlice.ts

```diff
@@ -122,6 +123,13 @@ window.electron.ipcRenderer.madara.onNodeLogs((event: any, data: string) => {
 // set up listener to set isRunning to false when node stops
 // and terminate all running apps
 window.electron.ipcRenderer.madara.onNodeStop(() => {
+  const runningApps = useAppSelector(selectRunningApps);
+
+  // Terminate all currently running apps
+  Object.keys(runningApps).forEach((appId) => {
+    window.electron.ipcRenderer.madaraApp.stopApp(appId);
+  });
+
```

Comment:
why are we adding this?

Reply:
sorry its a missed unresolved comflict from the previous, i have fixed it already